libs/components/core/src/index.ts

```diff
@@ -94,6 +94,9 @@ export { SkyResizeObserverService } from './lib/modules/resize-observer/resize-o
 
 export { SkyScreenReaderLabelDirective } from './lib/modules/screen-reader-label/screen-reader-label.directive';
 
+export { SkyScrollShadowDirective } from './lib/modules/scroll-shadow/scroll-shadow.directive';
```

Comment:
Needs a "lambda" export name, with a companion `SkyScrollShadowModule` that exports this directive.

Reply:
We talked offline and since it's "internal" we'll just take the risk with potential issues with the API.